bcl2fastq/index_summary.py

```diff
@@ -110,18 +143,24 @@ def update_pruned_unknown_hidden(run_alias):
     [dep.Input('sample_run_hidden', 'children')]
 )
 def update_known_index_bar(run_json):
+    """ When input (sample_run_hidden) is changed, function is rerun to get update bar graph of sample indices
+
+           Attributes:
+               run(list): name of run
+               data(list): empty list appended with values from "For" function
```

Comment:
Rather than describe the transitions of the variable ("empty list", "appended"), can you describe the final purpose of the variables? Maybe some thing like `object of run data for graphing the indices`. (Same with method below)